mw-webapp/src/hooks/useChangeTheme.ts

```diff
@@ -0,0 +1,20 @@
+import {useState} from "react";
+import {Theme, ThemeWorker} from "src/utils/ThemeWorker";
+
+/**
+ *Custom hook to change theme
+ */
+export const useChangeTheme = () => {
+  const currentTheme = ThemeWorker.getCurrentTheme();
+  const [theme, setTheme] = useState(currentTheme);
+
+  /**
+   *OnChangeTheme
+   */
+  const onChangeTheme = (value: string) => {
+    ThemeWorker.setTheme(value as Theme);
+    setTheme(value as Theme);
```

Comment:
Does it possible to avoid "as" operator?

Reply:
In this case, you can do this:
const onChangeTheme = (value: Theme) => {
    ThemeWorker.setTheme(value);
    setTheme(value);
};